lti_consumer/templates/html/student.html

```diff
@@ -1,18 +1,28 @@
-<h2 class="problem-header">
-    ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
-    ${display_name} (External resource)
-</h2>
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%page expression_filter="h"/>
```

Comment:
@chrisndodge: I appreciate you going for the safe template work on this.
At the bottom of this file, you'll find:
```
   ## sanitized with bleach in view
   ${comment}
```
I assume that this also takes HTML and will need the `| n, decode.utf8` filter.  Also, since this mentions sanitization and bleach, maybe you could add the same comment/view sanitization code to description to keep it safe as well?